make/firmware-arches.mk

```diff
@@ -11,7 +11,8 @@ else ifneq "$(findstring STM32F40,$(CHIP))" ""
 OPENOCD_JTAG_CONFIG ?= stlink-v2.cfg
 OPENOCD_CONFIG      := stm32f4x.cfg
 MCU                 := cortex-m4
-STM32_TYPE          := STM32F40_41xxx
+STM32_TYPE          := STM32F405xx
+STM32_SUBTYPE       := STM32F40_41xxx
```

Comment:
I've been looking at it some more, and it'd involve including ChibiOS' stm32_registry.h header in the different StdPeriphs. I'd figure an unsuitably named define in the makefile is the lesser evil option.

Reply:
Yea, no, trying to work stm32_registry.h into it creates more drama around the bootloaders and flightlib, considering that I somehow have to stick it into pios.h for some situations.